src/joysticks/ConsoleController.java

```diff
@@ -0,0 +1,448 @@
+     * Whether the button up was released since the last check.
+     *
+     * @return Whether the button was released since the last check.
+     */
+    public boolean getRawButtonUpReleased() {
+        return getRawButtonReleased(buttonUp);
+    }
+
+    /**
+     * Read the value of the center left button on the controller.
+     *
+     * @return The state of the button.
+     */
+    public boolean getRawCenterLeftButton() {
+        return getRawButton(centerLeft);
+    }
+
+    /**
+     * Whether the center left button button was pressed since the last check.
+     *
+     * @return Whether the button was pressed since the last check.
+     */
+    public boolean getRawCenterLeftButtonPressed() {
+        return getRawButtonPressed(centerLeft);
+    }
+
+    /**
+     * Whether the center left button was released since the last check.
+     *
+     * @return Whether the button was released since the last check.
+     */
+    public boolean getRawCenterLeftButtonReleased() {
+        return getRawButtonReleased(centerLeft);
+    }
+
+    /**
+     * Read the value of the center right button on the controller.
+     *
+     * @return The state of the button.
+     */
+    public boolean getRawCenterRightButton() {
+        return getRawButton(centerRight);
+    }
+
+    /**
+     * Whether the center right button was pressed since the last check.
+     *
+     * @return Whether the button was pressed since the last check.
+     */
+    public boolean getRawCenterRightButtonPressed() {
+        return getRawButtonPressed(centerRight);
+    }
+
+    /**
+     * Whether the center right button was released since the last check.
+     *
+     * @return Whether the button was released since the last check.
+     */
+    public boolean getRawCenterRightButtonReleased() {
+        return getRawButtonReleased(centerRight);
+    }
+
+    public int getButtonDown() {
+        return buttonDown;
+    }
+
+    public int getButtonUp() {
+        return buttonUp;
+    }
+
+    public int getButtonLeft() {
+        return buttonLeft;
+    }
+
+    public int getButtonRight() {
+        return buttonRight;
+    }
+
+    public int getCenterRight() {
+        return centerRight;
+    }
+
+    public int getCenterLeft() {
+        return centerLeft;
+    }
+
+    public int getStickLeft() {
+        return stickLeft;
+    }
+
+    public int getStickRight() {
+        return stickRight;
+    }
+
+    public int getAxisLeftX() {
+        return axisLeftX;
+    }
+
+    public int getAxisLeftY() {
+        return axisLeftY;
+    }
+
+    public int getAxisRightX() {
+        return axisRightX;
+    }
+
+    public int getAxisRightY() {
+        return axisRightY;
+    }
+
+    public int getLeftTrigger() {
+        return leftTrigger;
+    }
+
+    public int getRightTrigger() {
+        return rightTrigger;
+    }
+
+    public int getBumperRight() {
+        return bumperRight;
+    }
+
+    public int getBumperLeft() {
+        return bumperLeft;
```

Comment:
if u r very very very bored try to put in the same order as of the - raw butt funcs